src/GlobalStyle.tsx

```diff
@@ -1,26 +1,18 @@
 import { createGlobalStyle } from "styled-components";
 import theme from "./components/theme";
-import background from "./components/assets/backgrounds/background.png";
-
 
 const GlobalStyle = createGlobalStyle`
-@font-face{
-  font-family: SangBleu;
-  src: url(SangBleuOGSerif-Light.otf)
-}
-  html {
```

Comment:
So at the moment the `Div` tag of the `Background` component is not taking up 100% of the page

If you move across the site you start to notice the pattern. It's taking up 100% of its outer container which is only the size of its contents. 

Where you've deleted here the `html {
  height: 100%
} `  in the `GlobalStyle` component if you add it back in and then add `height: 100%` to the `Div` styled component like this: 

<img width="428" alt="Screenshot 2020-10-26 at 11 03 58" src="https://user-images.githubusercontent.com/53219789/97159318-f774ad80-177a-11eb-8c7b-de359eaf5abb.png">

then that should do the trick